cpp/ScanChain_Hgg.C

```diff
@@ -327,6 +325,38 @@ int ScanChain_Hgg(TChain *ch, double genEventSumw, TString year, TString process
       float weight = 1.0;
       if ( isMC ) {
         weight = nt.genWeight();
+      }
```

Comment:
Why change the position of the bracket? Now you are excluding from the `isMC` loop the application of the PU weight (which commented out for now but I am working on including it) and this leads to a logical error...

Reply:
Then please move the bracket to its initial position (there is no reason to move it in the first place).

Reply:
addressed